docs/operations/libraries/workflow_review_alpha.md

```diff
@@ -0,0 +1,24 @@
+## Library Module Alpha PR Review Workflow
+This describes the process for reviewing a PR to a `library` module `dev` branch.
+
+### Application
+This process applies when reviewing a PR to the `dev` branch of a `library` module.
+
+### Process
+- Review changes to imports and verify dependency minimum versions are accurate.
+- Ensure code changes have adequate unit test coverage.
+- Ensure there are no breaking changes.
```

Comment:
What happens if there are?

Reply:
Good call out. I should specify that anything not passing these checks should be noted in a review requesting changes